salesagility/SuiteCRM
# [FIX] installer copies (unused) sample image on each run in picture and sets non existing email field

## Motivation and Context

On each run of the installer, an image is created (copied) in the upload folder
## How To Test This

Run the installer and check for the created files in the upload directory (= image of a nice girl, with as filename a uuid)
## Types of changes
